src/aequitas_webapp/views.py

```diff
@@ -164,7 +164,7 @@ def audit_file(name, dirname):
                       attr_cols=group_variables)
 
     (_gv_df, report) = audit(df,
-                             model_id=1,
+                             # model_id=1,
```

Comment:
ditto comment

Reply:
Confirming this should not be passed in audit anymore!